cmd/execution_builder.go

```diff
@@ -400,77 +421,68 @@ func (e *ExecutionNodeBuilder) LoadComponentsAndModules() {
 
 			return compactor, nil
 		}).
-		Component("execution data service", func(node *NodeConfig) (module.ReadyDoneAware, error) {
-			err := os.MkdirAll(e.exeConf.executionDataDir, 0700)
-
+		Component("execution data pruner", func(node *NodeConfig) (module.ReadyDoneAware, error) {
```

Comment:
is there a way to disable pruning if I wanted to keep everything?

Reply:
No, you would just set the threshold very large